System/Secretary/DeskEventManager.ts

```diff
@@ -0,0 +1,63 @@
+/**
+ * Manages event handlers for an event aware class
+ */
+import { DeskEvent } from "./DeskEvent";
+
+class DeskEventItem {
+    id: number;
+    deskEvent: DeskEvent;
+    constructor(id: number, deskEvent: DeskEvent) {
+        this.id = id;
+        this.deskEvent = deskEvent;
+    }
+}
+
+class DeskEventInfo {
+    id: number;
+    constructor(id: number) {
+        this.id = id;
+    }
+}
```

Comment:
In TypeScript, things like this are often implemented as interfaces rather than as classes. If the goal is just to define the structure of a JS object, I'd use interfaces for these.

Reply:
```ts
interface DeskEventItem {
  id: number;
  deskEvent: DeskEvent
}

const item: DeskEventItem = {
  id: 3,
  deskEvent: new DeskEvent()
}
```